### PR TITLE
Ch2 - why R

### PR DESCRIPTION
**Pull request by carriewright11**
#### What changes are being implemented in this Pull Request?

filling out chapter 2 about why R is good for reproducibility 

### Tell potential reviewers what kind of feedback you are soliciting.

@cansavvy what do you think? I am happy to make any changes to the parrot you'd like - but we could have cartoons like this.

### New Content Checklist

- [X] New content/chapter contains learning objectives.

-  [X] Every new image has [alt text and is in a Google Slide](https://www.ottrproject.org/writing_content.html#Accessibility).

- [x] Each slide is described in the notes of the slide so learners relying on a screen reader can access the content. See https://lastcallmedia.com/blog/accessible-comics for more guidance on this.

- [x] The color palette choices of the slide are contrasted in a way that is friendly to those with color vision deficiencies.
You can check this using [Color Oracle](https://colororacle.org/).

### CODE REVIEW

File: 02-why-R.Rmd [hunk 1]
```diff
@@ -10,14 +10,115 @@ ottrpal::set_knitr_image_path()
...
+```
+
+
+The term open source means that the code is publicly available.
+Thus all of the code involved in creating R is actually publicly available! This enables users to check what code is used in a particular package (a set of code that allows you to do various things) so that they can modify or build upon the code if they would like to. 
+
+In fact, many users create their own R packages to share their code with others.  There are places such as the Comprehensive R Archive Network ([CRAN](https://cran.r-project.org/)) and elsewhere that allow users to publish their own packages for others to use. 
+
+<div class = "dictionary">
+- programming language - a specified set of notations to tell a computer what to do
```
Comment (cansavvy) [inline 02-why-R.Rmd]: It my be helpful for the terms to be in bold so we can follow along more easily.

File: 02-why-R.Rmd [hunk 2]
```diff
@@ -10,14 +10,115 @@ ottrpal::set_knitr_image_path()
...
+```
+
+This means that overall your code has a better chance of being more accessible than if it were written in a language that is not open source or that has limited support. 
+
+You can also find support to make sure your code does what you want it to, as well as support to make your code as reproducible as possible.
+
+There are several R community groups that are especially helpful:
+
+- [R Ladies](https://www.rladies.org/) - a support group that is not just for ladies, but is open to anyone who wants to improve their R skills! There are local chapters in many large cities that often have in-person meetings. 
+- There are lots of useful resources, such as the [R for Data Science book](https://r4ds.had.co.nz/) (written by two developers at Posit (formally called RStudio) which develops lots of core R packages), resources and online courses from the [Johns Hopkins Data Science Lab](https://jhudatascience.org/courses.html) including [Open Case Studies](https://www.opencasestudies.org/), resources and workshops from [Data Carpentry](https://datacarpentry.org/),  [Dataquest](https://www.dataquest.io/v2/), and more!
```
Comment (cansavvy) [inline 02-why-R.Rmd]:
```suggestion
- There are lots of useful resources, such as the [R for Data Science book](https://r4ds.had.co.nz/) (written by two developers at Posit (formally called RStudio) which develops lots of core R packages), resources and online courses from the [Johns Hopkins Data Science Lab](https://jhudatascience.org/courses.html) including [Open Case Studies](https://www.opencasestudies.org/), resources and workshops from [Data Carpentry](https://datacarpentry.org/),  [Dataquest](https://www.dataquest.io/v2/), [DataTrail](https://datatrail-jhu.github.io/DataTrail/) and more!
```
Don't mind if I do...

File: 02-why-R.Rmd [hunk 3]
```diff
@@ -10,14 +10,115 @@ ottrpal::set_knitr_image_path()
...
+See this [link](https://jhudatascience.org/intro_to_r/resources.html) for more R resources.
+
+## Designed for data
+
+R is a statistical programming language, meaning it was designed to help you analyze data. It is the main focus of the language. This is one of the major advantages of using R over other programming languages that have more general purposes. 
+
+Because of this many people have designed useful packages that are especially relevant to:
+
+1) Dealing with messy data in a systematic and reproducible way to get it into a state that is useful for data analysis
+2) Statistical analysis of data
```
Comment (cansavvy) [inline 02-why-R.Rmd]:
```suggestion
2) Producing statistical analysis of data
```
Just making it grammatically parallel to the first and third points.

File: 02-why-R.Rmd [hunk 4]
```diff
@@ -10,14 +10,115 @@ ottrpal::set_knitr_image_path()
...
+R can also import data from many different sources that other statistical software can't handle (including scraping data from websites or [PDFs](https://www.adobe.com/acrobat/about-adobe-pdf.html)). This allows users much more flexibility to use data as close to the source as possible. This can enable users to stop copy and pasting data and reduce the risk of human error. If you are interested, see [Open Case Studies](https://www.opencasestudies.org/) for more guidance on importing many different kinds of data. 
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "I created errors copying my data into Excel and spent hours figuring it out later! I’m glad R can help!"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g21c5ab757ec_0_78")
+```
+
+
+The following design features make R especially helpful for creating reproducible analyses:
+
+1. It enables users to work with messy data and get it ready for analysis
```
Comment (cansavvy) [inline 02-why-R.Rmd]: These points seem very similar to the points above but I guess that is okay.

File: 02-why-R.Rmd [hunk 5]
```diff
@@ -10,14 +10,115 @@ ottrpal::set_knitr_image_path()
...
+
+The following design features make R especially helpful for creating reproducible analyses:
+
+1. It enables users to work with messy data and get it ready for analysis 
+1. Users can create effective plots using the same program as for data prep and analysis
+1. It helps create reports that can show the entire data analysis process from importing the data to making plots
+1. It is easier to import data closer to the original source, rather than converting files or copy and pasting data, which can result in accidental modifications of the data 
+
+
+## Conclusion
```
Comment (cansavvy) [inline 02-why-R.Rmd:110]: It's handy for me to see your structure here. I'll try to make sure my chapters parallel this structure!

**Review by cansavvy — APPROVED** [review]
Some very eensie teensie comments but looks great! 1st chapter down!

**Comment by cansavvy** [comment]
Oh, you'll want to make sure if you merge #11 first, that you merge main to here so that you don't accidentally overwrite your style changes. 

The parrot is growing on me.

File: 02-why-R.Rmd (hunk 2 above)
Comment (carriewright11) [reply 02-why-R.Rmd]: :P no problem!

File: 02-why-R.Rmd (hunk 5 above)
Comment (carriewright11) [reply 02-why-R.Rmd:110]: cool!

**Comment by carriewright11** [comment]
> Oh, you'll want to make sure if you merge #11 first, that you merge main to here so that you don't accidentally overwrite your style changes.
> 
> The parrot is growing on me.

I am happy to make any changes you would like to him!

File: 02-why-R.Rmd (hunk 4 above)
Comment (carriewright11) [reply 02-why-R.Rmd]: hmm yes, I intended it as a section summary I can see if I could rework it somehow

File: 02-why-R.Rmd (hunk 1 above)
Comment (carriewright11) [reply 02-why-R.Rmd]: will do

File: 02-why-R.Rmd (hunk 4 above)
Comment (carriewright11) [reply 02-why-R.Rmd]: I made the summaries of how each section connects to the theme of reproducibility more clearly - so each section has a similar structure. I think that should help? I also added some info about specific packages/tools

**Comment by carriewright11** [comment]
@cansavvy mind checking my second-to-last commit to see if it improved?

**Review by cansavvy — APPROVED** [review]
Looks great! Let's merge it!

**Pull request merged**